src/grpc/infer_handler.cc

```diff
@@ -37,6 +37,37 @@ NextUniqueId()
 
 namespace triton { namespace server { namespace grpc {
 
+// Define operator in triton::server::grpc namespace for use in all helpers
```

Comment:
question: since we don't have this type of comment for others - would it be safe to remove?

Reply:
I added it because it got lost in the last restructure of these files, but I don't mind removing it - I doubt it would stop a similar thing in the future anyways.